module1.js

```diff
@@ -25,7 +33,14 @@ function question1(name1) {
 */
 
 function question2(number1, number2) {
-  // TODO:
+  {
+    if (Number.isInteger(number1 / number2)) {
```

Comment:
Hmm, this is not quite right. How do you test equality of two numbers in JS? How does this function do if you pass 10 and 5 for example?